Creature.playground/Contents.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+//Creation of Creature Structure
+
+struct Creature{
+    var name:String
+    var description:String
+    var isGood:Bool
+    var magicPower:Int
+    
+    //Initialisation of structure
+    
+    init(name: String, description: String, isGood: Bool, magicPower: Int) {
+        self.name = name
+        self.description = description
+        self.isGood = isGood
+        self.magicPower = magicPower
+    }
+    
+    // Computed Property
+    var ability:String
+    {
+        get{
+            return "\(self.name) special ability is \(fibonacciAbility(n: magicPower))"
+        }
+    }
+    
+    //Interaction of Creatures.
+    func interactWith(anotherInstance: Creature) -> String{
```

Comment:
Style Issues. A space is needed between the return type and the opening {. A space is also need between the colon and the type in your variable declarations.